### PR TITLE
gives sentinel the ability to swap to light acid spit

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
image
gives sentinel the ability to spit acid

## Why It's Good For The Game
sentinel is kinda a bad case right now, it'd be nice to make it a tad bit better, this gives it more versatility, i can reduce the damage if its too oppressive.

## Changelog
:cl:
balance: gives sentinel the ability to spit light acid
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -61. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by MLGTASTICa** [comment]
Isn't that acid spit as strong as the prae one ? also i personally feel like spitters are in a good spot for now , the only thing that needs to be fixed is their neurotoxin sting not doing anything honestly.

**Comment by Pariah919** [comment]
>Isn't that acid spit as strong as the prae one ? also i personally feel like spitters are in a good spot for now , the only thing that needs to be fixed is their neurotoxin sting not doing anything honestly.
>
No. It does 12 less damage. It also doesn't leave a splatter.
Sentinel themselves are kind of bad and I'd like to see how the changes go.

**Comment by MLGTASTICa** [comment]
> > Isn't that acid spit as strong as the prae one ? also i personally feel like spitters are in a good spot for now , the only thing that needs to be fixed is their neurotoxin sting not doing anything honestly.
> 
> No. It does 12 less damage. It also doesn't leave a splatter.
> Sentinel themselves are kind of bad and I'd like to see how the changes go.

Sentinels are not bad . If you land 5 shots on a marine they WILL fall asleep leaving them very vurnerable. You can also melle slash and spam spits , which can crit a marine very quickyl. They are also a T1

**Comment by jroinc** [comment]
> Sentinels are not bad . If you land 5 shots on a marine they WILL fall asleep leaving them very vurnerable. You can also melle slash and spam spits , which can crit a marine very quickyl. They are also a T1.

Alternatively, if you DON'T land 5 shots IN A ROW, within 3 seconds or less of each other, they won't go down (they don't go to sleep, they just lose their stamina), and they'll heal off 1 shot every 4 seconds.

I'm in support of giving their spit a slight amount of damage (per above), just so that they aren't 100% useless when dealing with more than one guy.

**Comment by MLGTASTICa** [comment]
> > Sentinels are not bad . If you land 5 shots on a marine they WILL fall asleep leaving them very vurnerable. You can also melle slash and spam spits , which can crit a marine very quickyl. They are also a T1.
> 
> Alternatively, if you DON'T land 5 shots IN A ROW, within 3 seconds or less of each other, they won't go down (they don't go to sleep, they just lose their stamina), and they'll heal off 1 shot every 4 seconds.
> 
> I'm in support of giving their spit a slight amount of damage (per above), just so that they aren't 100% useless when dealing with more than one guy.

Then the spit itself should be buffed to have long-term consequences on max stamina

**Review by Hughgent — APPROVED** [review]

**Comment by Escamado** [comment]
Also theres a bug that the button for switching spit types gets stuck after aging, I couldnt change it anymore after being mature,elder and ancient.

File: code/modules/mob/living/carbon/xenomorph/castes/sentinel/castedatum_sentinel.dm [hunk 1]
```diff
@@ -37,13 +37,14 @@
 
 	// *** Ranged Attack *** //
 	spit_delay = 1.3 SECONDS
-	spit_types = list(/datum/ammo/xeno/toxin)
+	spit_types = list(/datum/ammo/xeno/toxin, /datum/ammo/xeno/acid)
```
Comment (Hughgent) [inline code/modules/mob/living/carbon/xenomorph/castes/sentinel/castedatum_sentinel.dm:40]: This needs to be replicated for all maturities.

**Review by Hughgent — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/sentinel/castedatum_sentinel.dm (hunk 1 above)
Comment (Pariah919) [reply code/modules/mob/living/carbon/xenomorph/castes/sentinel/castedatum_sentinel.dm:40]: ok

**Review by Hughgent — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-71). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**